zcash_proofs/src/sapling/verifier/batch.rs

```diff
@@ -47,22 +47,22 @@ impl BatchValidator {
 
         let mut ctx = SaplingVerificationContextInner::new();
 
-        for spend in bundle.shielded_spends {
+        for spend in bundle.shielded_spends() {
```

Comment:
This code not being covered by tests is concerning. (I know that is a preexisting problem.)